src/api/rule-engine/Rule.ts

```diff
@@ -299,10 +300,10 @@ export class RuleService extends CoreWebService {
     });
   }
 
-  loadRules():Observable<RuleModel[]|CwError> {
+  loadRules( site:Site ):Observable<RuleModel[]|CwError> {
```

Comment:
Why do we have to send the site? shouldn't we have this somehow globally, because send the site every time we are doing a request will sucks.